app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -269,7 +269,7 @@ class ThreadListFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
             SwipeAction.QUICKACTIONS_MENU -> {
                 safeNavigate(
                     ThreadListFragmentDirections.actionThreadListFragmentToThreadActionsBottomSheetDialog(
-                        messageUidToReplyTo = MessageController.getMessageUidToReplyTo(thread.messages),
+                        messageUidToReplyTo = MessageController.getMessageToReplyTo(thread.messages).uid,
```

Comment:
it should not be done here, but directly where it is needed, i.e. in the destination fragment.